src/hotspot/cpu/s390/vm_version_s390.cpp

```diff
@@ -285,27 +296,27 @@ void VM_Version::set_features_string() {
     ambiguity++;
   }
   if (is_z196()) {
-    _features_string = "system-z, g5-z196, ldisp_fast, extimm, pcrel_load/store, cmpb, cond_load/store, interlocked_update";
+    _features_string = "system-z, g5-z196, ldisp_fast, extimm, pcrel_load/store, cmpb, cond_load/store, interlocked_update, out-of-support_as_of_2021-12-31";
     _model_string = z_name[5];
     ambiguity++;
   }
   if (is_z10()) {
-    _features_string = "system-z, g4-z10, ldisp_fast, extimm, pcrel_load/store, cmpb";
+    _features_string = "system-z, g4-z10, ldisp_fast, extimm, pcrel_load/store, cmpb, out-of-support_as_of_2019-12-31";
     _model_string = z_name[4];
     ambiguity++;
   }
   if (is_z9()) {
-    _features_string = "system-z, g3-z9, ldisp_fast, extimm, out-of-support as of 2016-04-01";
+    _features_string = "system-z, g3-z9, ldisp_fast, extimm, out-of-support_as_of_2016-04-01";
```

Comment:
How does this relate to the table above?

Reply:
Changed such that End-of-Support information is taken from table z_EOS[].